### DISCUSSION TITLE
Using NVDA's say all in Bookworm fails to make system cursor and review cursor follow

### DISCUSSION THREAD

**Discussion by cary-rowen** [mush42/bookworm#116] [General]
Hi all

I try to Say all text by pressing NVDA plus down arrow or numpad plus,
NVDA speak correctly, but neither the system cursor nor the review cursor move with it.

thanks

**Comment by mush42** [comment]
Hello @cary-rowen 
I think this is the default behavior in nvda. When using narrator, the system cursor and the review cursor move by character. The cursor position behavior depends on the screen reader. Bookworm uses the standard rich text control on Windows.

**Comment by cary-rowen** [comment]
Hi @mush42 

NVDA seems to make system cursor and review cursor move in other rich text controls,
Bookworm is a particular example I've found so far. Does anyone else have any suggestions, or do we need to report this behavior to NVAcces.

Thanks

**Comment by cary-rowen** [comment]
Hi, @mush42 

Another problem I found was that NVDA was unable to move the system cursor to the review cursor position by pressing twice nvda+shift+numpad minus.

Thanks

**Comment by mush42** [comment]
@cary-rowen 

Did you try testing this behavior with NVDA's log viewer?

Base on my testing, this issue also exists with NVDA's log viewer.

We're using the same GUI toolkit as NVDA, so please check if you can replicate this issue there as well.

Best
Musharraf

**Comment by cary-rowen** [comment]
Hi @mush42 
You are right, I found the same behavior on the NVDA log viewer, I have created an issue in the NVDA repository.
[it's here](https://github.com/nvaccess/nvda/issues/13420
it's here)

Thanks
